amps/ags/rm-community/rm-community-rest-api-explorer/src/main/webapp/definitions/gs-core-api.yaml

```diff
@@ -2092,6 +2212,248 @@ paths:
+          schema:
+            $ref: '#/definitions/HoldModelEntry'
+        '400':
+          description: |
+            Invalid parameter: **holdId** is not a valid format
+        '401':
+          description: Authentication failed
+        '403':
+          description: Current user does not have permission to read **holdId**
+        '404':
+          description: "**holdId** does not exist"
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+    put:
+      tags:
+        - holds
+      summary: Update a hold
+      description: |
+        Updates the hold with id **holdId**. For example, you can rename a hold:
+        ```JSON
+        {
+          "name":"My new name"
+        }
+        ```
+      operationId: updateHold
+      parameters:
+        - $ref: '#/parameters/holdIdParam'
+        - in: body
+          name: holdBodyUpdate
+          description: The hold information to update.
+          required: true
+          schema:
+            $ref: '#/definitions/HoldUpdateBodyModel'
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/HoldModelEntry'
+        '400':
+          description: |
+            Invalid parameter: the update request is invalid or **holdId** is not a valid format or **holdBodyUpdate** is invalid
+        '401':
+          description: Authentication failed
+        '403':
+          description: Current user does not have permission to update **holdId**
+        '404':
+          description: "**holdId** does not exist"
+        '409':
+          description: Updated name clashes with an existing node in the current parent folder
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+    delete:
+      tags:
+        - holds
+      summary: Delete a hold
+      description: |
+        Deletes the hold with id **holdId**.
+      operationId: deleteHold
+      parameters:
+        - $ref: '#/parameters/holdIdParam'
+        - $ref: '#/parameters/reasonParam'
+      produces:
+        - application/json
+      responses:
+        '204':
+          description: Successful response
+        '400':
+          description: |
+            Invalid parameter: **holdId** is not a valid format
+        '401':
+          description: Authentication failed
+        '403':
+          description: Current user does not have permission to delete **holdId**
+        '404':
+          description: "**holdId** does not exist"
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+  '/holds/{holdId}/children':
+    post:
+      tags:
+        - holds
+      summary: Add children to a hold
+      description: |
+        Add a child of a hold with id **holdId**.
+
+        You must specify the child **id**.
```

Comment:
I think we should specify what happens if the child is already a child of the hold. I don't mind whether we go with a 201 success or a 409 failure.

Reply:
Done.